theme/less/_homepage.less

```diff
@@ -0,0 +1,95 @@
+.homepage div
+{
+  grid-template-areas: 
+    "header header"
+    "main main"
+    "footer footer";
+}
+
+.homepage main article
+{
+  display: grid;
+  grid-template-columns: auto auto;
+}
+
+.homepage main article > :first-child
+{
+  grid-column: 1 e("/") 3;
+}
+
+.box
+{
+  position: relative;
```

Comment:
As we are now introducing possitioning we might need a `z-index` for our header menu items, as this currently is overflowing the top menu absolute positioning.